project/SparkBuild.scala

```diff
@@ -59,7 +59,7 @@ object BuildCommons {
   ) = Seq(
     "core", "graphx", "mllib", "mllib-local", "repl", "network-common", "network-shuffle", "launcher", "unsafe",
     "tags", "sketch", "kvstore"
-  ).map(ProjectRef(buildLocation, _)) ++ sqlProjects ++ streamingProjects ++ Seq(connect)
+  ).map(ProjectRef(buildLocation, _)) ++ sqlProjects ++ streamingProjects ++ Seq(connect) ++ Seq(protobuf)
```

Comment:
This change is not needed right? (noticed this while fixing merge conflicts for Databricks's own repo).

Reply:
I mean protobuf is already included in `sqlProjeccts`.

Reply:
@rangadi yes looks like this change is not needed. will make change this followup PR